src/__tests__/App.test.tsx

```diff
@@ -63,6 +63,7 @@ const playQueueState: PlayQueue = {
   entry: [],
   shuffledEntry: [],
   sortedEntry: [],
+  resume: false,
```

Comment:
This property is probably unneeded in redux since it's not being referenced anywhere. The main process is just referencing the settings value instead of this in the state.